lib/karafka/active_job/consumer.rb

```diff
@@ -5,20 +5,91 @@ module ActiveJob
     # This is the consumer for ActiveJob that eats the messages enqueued with it one after another.
     # It marks the offset after each message, so we make sure, none of the jobs is executed twice
     class Consumer < ::Karafka::BaseConsumer
+      # Defaults for consuming
+      # The can be updated by using `#karafka_options` on the job
+      DEFAULTS = {
+        timeout: 300000,          # (5 minutes)
```

Comment:
I set this to the default `max.poll.interval.ms` to keep consistency with the current implementation.